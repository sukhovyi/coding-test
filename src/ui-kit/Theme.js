const Theme = Object.freeze({
  backgroundColor: '#FFFFFF',
  foregroundColor: '#333333',
  colors: {
    primary: '#0063D9',
    secondary: '#E7E7E7',
    tertiary: '#AAAAAA',
    danger: '#CC2E2E',
    success: '#3FBA69',
  },
  fontSize: '16px',
});

export default Theme;
