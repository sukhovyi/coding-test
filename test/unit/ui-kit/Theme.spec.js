import Theme from '../../../src/ui-kit/Theme';

describe('ui-kit/Theme', () => {
  it('should have correct setup', () => {
    expect(Theme).toMatchSnapshot();
  });
});
