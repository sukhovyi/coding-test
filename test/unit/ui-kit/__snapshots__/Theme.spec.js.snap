// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-kit/Theme should have correct setup 1`] = `
Object {
  "backgroundColor": "#FFFFFF",
  "colors": Object {
    "danger": "#CC2E2E",
    "primary": "#0063D9",
    "secondary": "#E7E7E7",
    "success": "#3FBA69",
    "tertiary": "#AAAAAA",
  },
  "fontSize": "16px",
  "foregroundColor": "#333333",
}
`;
